brainframe/api/stubs/capsules.py

```diff
@@ -28,6 +29,38 @@ def get_capsules(self, timeout=DEFAULT_TIMEOUT) -> List[Capsule]:
         capsules, _ = self._get_json(req, timeout)
         return [Capsule.from_dict(d) for d in capsules]
 
+    def set_capsule(self, storage_id: int,
+                    source_path: Optional[Path] = None,
+                    timeout=DEFAULT_TIMEOUT) -> Capsule:
+        """Sends capsule data to the server to be loaded and initialized.
+
+        :param storage_id: The ID of the raw capsule data
+        :param source_path: The path to the source code on the development
+            machine, if available. Providing this allows stack traces to point
+            to the correct source location.
```

Comment:
Consider adding: 
The source location is assumed to be a valid path on the server machine.

Reply:
Actually it's the opposite! The source location is expected to be a path on the developer's machine. That could be the same as the server machine, or not. The purpose is to allow the server to report exceptions to the developer that make sense from their perspective.

Reply:
> if available

What this means might not be clear enough. It might be inferred that _if_ the path is available, then it's required. I think it would be better to be a bit more explicit here:
```
:param source_path: If available, the path to the source code on the development
    machine can be provided. Doing so allows stack traces to point to the correct
    source location.
```

However this changes the "descriptive" pattern we're applying to each parameter, so maybe there's a better way to convey this